ArcGISRuntimeSDKQt_QMLSamples/CloudAndPortal/IntegratedWindowsAuthentication/IntegratedWindowsAuthentication.qml

```diff
@@ -0,0 +1,241 @@
+                Layout.margins: 2
+                spacing: 3
+
+                Button {
+                    text: qsTr("Search Public")
+                    onClicked: {
+                        searchPortal(arcgis_url, false);
+                    }
+                }
+                Button {
+                    text: qsTr("Search Secure")
+                    onClicked: {
+                        if (securePortalUrl.text) {
+                            searchPortal(securePortalUrl.text, forceLoginBox.checked);
+                        } else {
+                            webMapMsg.text = "Portal URL is empty. Please enter a portal URL"
+                            webMapMsg.visible = true;
+                            return;
+                        }
+                    }
+                }
+            }
+
+            ComboBox {
+                id:webmapsList
+                Layout.margins: 2
+                Layout.fillWidth: true
+                model: null
+            }
+
+            Button {
+                text: qsTr("Load Web Map")
+                Layout.fillWidth: true
+                Layout.margins: 2
+                onClicked: {
+                    if (webmapsList.currentIndex >= 0) {
+                        indicator.running = true;
+                        loadSelectedWebmap(portalItemListModel.get(webmapsList.currentIndex));
+                    }
+                }
+            }
+        }
+    }
+
+    BusyIndicator {
+        id: indicator
+        anchors.centerIn: parent
+        running: false
+    }
+
+    PortalQueryParametersForItems {
+        id: webmapQuery
+        types: [ Enums.PortalItemTypeWebMap ]
+    }
+
+    function searchPortal (portalUrl, forceLogin) {
+
+        var portal;
+        if (forceLogin) {
+            portal = ArcGISRuntimeEnvironment.createObject("Portal", {url: portalUrl, loginRequired: forceLogin});
+        } else {
+            portal = ArcGISRuntimeEnvironment.createObject("Portal", {url: portalUrl});
+        }
+
+        portal.loadStatusChanged.connect(function() {
+            if (portal.loadStatus === Enums.LoadStatusFailedToLoad) {
+                webMapMsg.text = portal.loadError.message;
+                webMapMsg.visible = true;
+                indicator.running = false;
+                return;
+            }
+
+            if (portal.loadStatus === Enums.LoadStatusLoaded){
+                portal.findItems(webmapQuery);
+                return;
+            }
+
+        });
+
+        portal.findItemsStatusChanged.connect(function() {
+            if ( portal.findItemsStatus === Enums.TaskStatusCompleted ) {
+                indicator.running = false;
+                portalItemListModel = portal.findItemsResult.itemResults;
+                var index = 0
+                var error = portalItemListModel.forEach(function(prtlItem) {
+                    listModelForComboBox[index] = prtlItem.title;
```

Comment:
no need to create a `listModelForComboBox` - the itemResults already returns you a list model that should be usable from within a combo box. You can access it via the title role. ComboBox allows you to set the role you want - https://doc.qt.io/archives/qt-5.11/qml-qtquick-controls2-combobox.html#textRole-prop